File: Conceitos de LPs/Tipos de Dados/Cadeia de Caracteres/CadeiaCaracteres.pas
//Pascal
program CadeiaCaracteres;

var
    str1, str2, concatenado, sub: string;
    iguais: boolean;

begin
    str1 := 'Ola';
    str2 := 'Mundo';

    // Concatenar
    concatenado := str1 + ' ' + str2;
    writeln('Concatenado: ', concatenado);

    // Substring
    sub := Copy(concatenado, 1, 3);
    writeln('Substring: ', sub);

    // Comparar strings
    iguais := str1 = str2;
    writeln('As strings são iguais? ', iguais);

    // Tamanho da string
    writeln('Tamanho de str1: ', Length(str1));
end.

File: Conceitos de LPs/Tipos de Dados/Cadeia de Caracteres/cadeiacaracteres.kt
//Kotlin
fun main() {
    val str1 = "Olá"
    val str2 = "Mundo"

    // Concatenar
    val concatenado = "$str1 $str2"
    println("Concatenado: $concatenado")

    // Substring
    val sub = concatenado.substring(0, 3)
    println("Substring: $sub")

    // Comparar strings
    val iguais = str1 == str2
    println("As strings são iguais? $iguais")

    // Tamanho da string
    println("Tamanho de str1: ${str1.length}")
}
